core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlEvaluateResult.java

```diff
@@ -66,10 +66,13 @@ protected boolean executeAction() throws InterruptedException {
         boolean hasResult = convertHasResult(getExpectedResult().getValue());
         String connectionName = convertConnectionName(getConnectionName().getValue());
 
-        Connection connection = ConnectionConfiguration.getInstance()
+        Connection connection;
+        connection = ConnectionConfiguration.getInstance()
                 .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
-                .get();
+                .orElseThrow(() -> new RuntimeException("Unknown connection name"));
```

Comment:
add 'connectionName' to the Exception message: cannot find connection 'connectionName'. This will help users during debugging

Reply:
done in next commit